app/src/androidTest/java/com/github/onlynotesswent/ui/overview/EditNoteTest.kt

```diff
@@ -76,6 +78,22 @@ class EditNoteTest {
     org.mockito.kotlin.verify(navigationActions, never()).navigateTo(Screen.OVERVIEW)
   }
 
+  @Test
+  fun clickAddCommentButton() {
```

Comment:
Here you could also test that when adding a new comment with `performTextInput()` it is correctly stored or displayed. Also, editing this comment could tested for instance by editing it and then asserting that the old version of the comment is different from the new one.